appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/properties/YoungAndroidNavigationMethodChoicePropertyEditor.java

```diff
@@ -0,0 +1,29 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2009-2011 Google, All Rights reserved
+// Copyright 2011-2012 MIT, All rights reserved
+// Released under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+package com.google.appinventor.client.editor.youngandroid.properties;
+
+import com.google.appinventor.client.widgets.properties.ChoicePropertyEditor;
+import com.google.appinventor.components.common.ComponentConstants;
+
+// cludge for now.  mimic how this is done for button shape
+
+
+/**
+ * Property editor for vertical alignment choice.
+ */
+public class YoungAndroidNavigationMethodChoicePropertyEditor extends ChoicePropertyEditor {
+
+  private static final Choice[] methods = new Choice[] {
+    new Choice("foot-walking", "foot-walking"),
```

Comment:
The first argument of each of these should be put into OdeMessages.java and called here to allow for internationalization of the text.